uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -526,6 +530,36 @@ function _tasksAssignments_allow_only_add_timeline_action(phpQueryObject $doc) {
   $actionDropdown->remove();
 }
 
+/**
+ * Adds the category form field to the case type edit screen.
+ *
+ * @param phpQueryObject $doc
+ */
+function _tasksAssignments_add_case_type_category_form_field (phpQueryObject $doc) {
+  $smarty = CRM_Core_Smarty::singleton();
+  $formFieldTemplatePath = realpath(__DIR__ . '/views/crm/case-type-category-form-field.tpl');
+
+  $caseTypeCategoryFieldId = CRM_Core_BAO_CustomField::getCustomFieldID(
+    'category',
+    'case_type_category'
+  );
+  $caseTypeCategoryOptions = civicrm_api3('OptionValue', 'get', [
```

Comment:
This can be improved with setting of returned value since only the `value` and `label` is used
```
'return' => ['value', 'label'],
```